massdriver-application-aws-ecs-service/main.tf

```diff
@@ -1,31 +1,29 @@
 locals {
+  region             = var.ecs_cluster.specs.aws.region
   vpc_id             = element(split("/", var.ecs_cluster.data.infrastructure.vpc.data.infrastructure.arn), 1)
   private_subnet_ids = toset([for subnet in var.ecs_cluster.data.infrastructure.vpc.data.infrastructure.private_subnets : element(split("/", subnet["arn"]), 1)])
   ecs_cluster_arn    = var.ecs_cluster.data.infrastructure.arn
+  ecs_cluster_name   = element(split("/", var.ecs_cluster.data.infrastructure.arn), 1)
 }
 
 module "application" {
-  source  = "github.com/massdriver-cloud/terraform-modules//massdriver-application?ref=fc5f7b1"
+  source  = "github.com/massdriver-cloud/terraform-modules//massdriver-application?ref=9d5ec14"
   name    = var.md_metadata.name_prefix
-  service = "function"
+  service = "container"
```

Comment:
uhh I don't think this is a valid type. I'll check but I think they are

- kubernetes
- function
- vm

Reply:
Yarp, looks like we don't have a `container`. 

https://github.com/massdriver-cloud/terraform-modules/blob/main/massdriver-application/main.tf#L32-L34

Functionally, `vm` and `function` are no different ATM. They are the same, and then `kubernetes` is different because of workload identity.

Reply:
LOL eff me. I see it now :)
https://github.com/massdriver-cloud/terraform-modules/pull/148/files#diff-1518749db33e83ba0f8a50ea99f8826c3620d6fde75f0519f1ba4400ce024533R4